src/App.vue

```diff
@@ -26,14 +25,19 @@ export default {
   },
   methods: {
     addItem(value) {
+      if(value == "") return;
       this.items.push(value);
     },
+    removeItem(itemID) {
+      this.items = this.items.filter((item) => {
+        return item.id != itemID;
+      })
```

Comment:
`this.items = this.items.filter(item => item.id != itemID)`

Reply:
done